src/node.cpp

```diff
@@ -984,6 +984,33 @@ NodeCounter Node::subnodeCounts() const
     return nc;
 }
 
+#ifdef ENABLE_SYNC
+void Node::setSyncable(const bool syncable, const bool notify)
+{
+    if (syncable != mSyncable)
+    {
+        mSyncable = syncable;
+        if (notify)
```

Comment:
I think we would want to always keep the node flag and the map and the db table all exactly 'in sync'?  thanks

Reply:
Will fix.